Makefile

```diff
@@ -477,44 +477,50 @@ $(LOCALBIN):
 
 ## Tools
 KUSTOMIZE ?= $(LOCALBIN)/kustomize
-# same as used in kubectl v1.28.x ; see https://github.com/kubernetes-sigs/kustomize/tree/master?tab=readme-ov-file#kubectl-integration
+
+# https://github.com/kubernetes/kubectl/blob/release-1.30/go.mod#L44
+# same as used in kubectl v1.30.x; https://github.com/kubernetes/kubectl/blob/release-1.30/go.mod#L44
 KUSTOMIZE_VERSION ?= v5.1.1
 
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 # List of available versions: https://storage.googleapis.com/kubebuilder-tools
-ENVTEST_K8S_VERSION = 1.28.3
+ENVTEST_K8S_VERSION = 1.30.0
 
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 # We went ahead of k8s 1.28 with controller-tools v0.14.0 (which depends on k8s 1.29) to unblock go 1.22 upgrade.
 # It should be *relatively* safe as the only changes from this controller-tools version are description in CRD.
 # Once we upgrade to k8s 1.29, there's no need to change CONTROLLER_TOOLS_VERSION, and this 3 lines can be removed.
 #
 # k8s deps @ 1.29.0 https://github.com/kubernetes-sigs/controller-tools/blob/<version>/go.mod
-CONTROLLER_TOOLS_VERSION ?= v0.14.0
+CONTROLLER_TOOLS_VERSION ?= v0.15.0
 
-CODE_GENERATOR_VERSION ?= v0.28.12
+# Should match the kubeernetes minor vesion
+CODE_GENERATOR_VERSION ?= v0.30.7
```

Comment:
Can you elaborate on why it should match the k8s minor version?

(also: typo: `s/kubeernetes/kubernetes/` :smile:)

Reply:
I actually don't know why it should match, though IIRC it was because of dep resolution errors

Reply:
Hm, if we're not sure, then let's not add this comment?